### PR TITLE
Create dotnet-core-2-2.md

### PR DESCRIPTION
**Pull request by yang-xiaodong**
Translate [`What's new in .NET Core 2.2`](https://github.com/dotnet/docs/blob/master/docs/core/whats-new/dotnet-core-2-2.md)

### CODE REVIEW

File: docs/core/whats-new/dotnet-core-2-2.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+从 .NET Core 2.2 开始，您可以在运行应用程序的 Main 方法之前使用 startup hook 来注入代码。 startup hook 使 host 可以在部署应用程序后自定义应用程序的行为，而无需重新编译或更改应用程序。
+
+我们希望托管服务提供商来定义自定义的配置和策略，包括可能影响主入口点的加载行为的设置，例如 <xref:System.Runtime.Loader.AssemblyLoadContext?displayProperty=nameWithType> 的行为。 hook 可用于设置跟踪或注入遥测，设置回调以进行处理，或定义其他依赖于环境的行为。 Hook 与入口点是分开的，因此不需要修改用户代码。
+
+参见 [Host startup hook](https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/host-startup-hook.md) 了解更多信息。
+
+## 请参阅
+
+* [.NET Core 的新增功能](index.md)
+* [.NET Core 2.2 的新增功能](/aspnet/core/release-notes/aspnetcore-2.2)
```
Comment (imba-tjd) [inline docs/core/whats-new/dotnet-core-2-2.md]: ASP.NET Core

**Comment by v-radelg** [comment]
Hello @yang-xiaodong  , thank for your contribution. We are currently reviewing it. 

We will communicate the decision taken accordingly. Thanks for your time and support.  

Kind regards,  Microsoft C&E Localization Team

**Comment by v-radelg** [comment]
Hello, @yang-xiaodong  ,  Thank you for your feedback and suggestion. 

The English source file has been updated recently. Your feedback was considered when localizing the new version.  

Many thanks for your contribution.  

Kind regards,  Microsoft C&E Localization Team

**Comment by yang-xiaodong** [comment]
@v-radelg  All right, so how should I handle this PR? I found some conflicts, so should I close this PR or try to resolve the conflict and make it can be merge?

**Comment by v-radelg** [comment]
@yang-xiaodong Thank you for your message. We just fixed the conflicts and merged the pr.

Kind regards, Microsoft C&E Localization Team

**Pull request merged**